tools/wiki_entities/util.py

```diff
@@ -0,0 +1,88 @@
+import codecs
+import re
+from mafan import simplify
+
+p = re.compile('^Q[0-9]+:.*$')
```

Comment:
nit: rename the variable to be more readable such as NEW_VERSION_PREFIX_REGEXP

Reply:
Done :smile: